### DISCUSSION TITLE
Filter by repository weirdness

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#118743] [Mobile] [answered]
Here we are in the mobile app. We are looking for issues and we decide to filter by repository. 

We get into this screen and we pick the repository and then we have two choices on the top you see.

One is an x, one is an arrow. 

No matter which one we pick we still need to pick it again or something, to get finally back to the filtered issues. 

Maybe it has something to do with this being intended as a overlay screen. 

But it occupies the entire cell phone so maybe that intention is misguided or something. 

It's hard to tell you the exact steps to reproduce. All I know is something funny happens. 

![Screenshot_20240412_063958_GitHub.jpg](https://github.com/community/community/assets/1230959/80d5ca9f-a257-4e18-82c8-758f0943f6ad)

<hr>

Okay I figured it out! But this is not to let GitHub off the hook though.

- We are not supposed to hit the x
- we are not supposed to hit the arrow 

It turns out we are supposed to... Drum roll.. 

Swipe downwards!

Okay but you can't blame the user for not figuring that out on their own. It takes some accidental swiping of the page to finally find the answer!

**Comment by stevepopovich** [comment]
hi @jidanni bit confused by your report here. The `X` button should just clear your search. The arrow should close the sheet. If you have a selection made, neither buttons should clear that selection. Swiping down should do the same thing as the arrow.

**Reply by jidanni** [reply, marked as answer]
Thanks. Seems I need some time to practice how to use this page properly...
